Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM python:3.5-slim-stretch
+FROM python:3.5-slim-buster
 
 # ensure unoconv can locate the uno library
-ENV PYTHONPATH=/usr/lib/python3/dist-packages
+ENV PYTHONPATH /usr/lib/python3/dist-packages
```

Comment:
I was wondering whether this was a bug but learned that bot (w/ or w/o `=`) are valid Dockerfile statement for setting an `ENV`: https://docs.docker.com/engine/reference/builder/#env.